web/migrations/0050_achievement_badge_type_achievement_teacher_and_more.py

```
@@ -0,0 +1,69 @@
+                ("minted_at", models.DateTimeField(blank=True, null=True)),
+                ("wallet_address", models.CharField(blank=True, max_length=42)),
+                ("icon_url", models.URLField(blank=True)),
+                (
+                    "achievement",
+                    models.OneToOneField(
+                        on_delete=django.db.models.deletion.CASCADE, related_name="nft_badge", to="web.achievement"
+                    ),
+                ),
+            ],
+        ),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)